tests/integration/telemetry/tracing/otelcollector/tracing_test.go

```diff
@@ -187,6 +184,13 @@ meshConfig:
         headers:
           - name: "some-header"
             value: "some-value"
+  - name: test-otel-res-detectors
+    opentelemetry:
+      service: opentelemetry-collector.istio-system.svc.cluster.local
+      port: 4317
+      resource_detectors:
+        environment: {}
+        dynatrace: {}
```

Comment:
is this line meaningful?

Reply:
I wanted to have a test that has all detectors, to make sure things work still. But if you prefer I can remove the Dynatrace one and leave only the environment.